tests/performance/custom_tld.xml

```diff
@@ -0,0 +1,11 @@
+<test>
+    <preconditions>
+        <table_exists>hits_100m_single</table_exists>
+    </preconditions>
+
+    <query>SELECT count() FROM hits_100m_single WHERE NOT ignore(firstSignificantSubdomain(URL))</query>
```

Comment:
Maybe we can put it into `first_significant_subdomain` or `url_hits`? Probably the latter, and the former can be deleted. For the perf tests, having one relatively big test with lots of related functions is often preferrable, because it's more convenient to compare the results, and the aggregate metric over the entire test is meaningful to some extent, e.g. 'how much faster is working with URLs'. Also we only run a subset of queries per test in PRs (no more than 15 queries IIRC), so adding more queries to a big test doesn't make it slower for PRs at least, only for master where we run everything.

Reply:
>Maybe we can put it into first_significant_subdomain or url_hits? Probably the latter, and the former can be deleted. 

Sure. So are you suggesting to move custom_tld.xml and first_significant_subdomain.xml to the url_hits.xml and remove old files?

>For the perf tests, having one relatively big test with lots of related functions is often preferrable, because it's more convenient to compare the results, and the aggregate metric over the entire test is meaningful to some extent, e.g. 'how much faster is working with URLs'

I guess it is better for seeing the picture "in general"

> Also we only run a subset of queries per test in PRs (no more than 15 queries IIRC), so adding more queries to a big test doesn't make it slower for PRs at least, only for master where we run everything.

But this means that new functions will not be even checked in performance tests? Then maybe it will be better to merge them in a separate PR? (since currently this one contains real numbers for those functions, while after it may even skip them, since url_hits.xml already contains 15+ queries)

Reply:
> Sure. So are you suggesting to move custom_tld.xml and first_significant_subdomain.xml to the url_hits.xml and remove old files?

Yep.

> But this means that new functions will not be even checked in performance tests?

This logic is disabled when the PR changes the test files, and we run all the queries.

Reply:
>This logic is disabled when the PR changes the test files, and we run all the queries.

Ok, then let's merge those